docs/installation.md

```diff
@@ -45,6 +45,12 @@ There are also two installation sources:
     pip install -e ".[all]"
     ```
 
+!!! note
+
+    Make sure you are viewing the version of the docs that matches your `syntheseus` installation.
+    Select [stable](https://microsoft.github.io/syntheseus/stable/) (0.3.0) if using the latest version available via `pip`,
```

Comment:
Mentioning `0.3.0` here as otherwise someone might be confused seeing how `stable` is not an option in the version selector. A downside is that this will have to be updated with each release (or we could figure out an option to automatically show what the latest version is here, but this may require plugging in a custom mkdocs extension).

Reply:
We could simply not explicitly state 0.3.0 and just write something like:

> Make sure you are viewing the version of the docs that matches your `syntheseus` installation.
> This will generally be a version tag (e.g. 0.X.Y). Select *stable* if you are using the latest version available from pip, or *dev* if installing directly from GitHub.

Would that be a reasonable solution?

Reply:
Except "select stable" doesn't make sense, because that is not present in the version selector? But I could rephrase to avoid saying the word "stable" entirely

Reply:
Ah true. That's weird, since the link `https://microsoft.github.io/syntheseus/stable/` works and points to the correct version. We could just say "select the correct version" or dev and leave it to the user then?

Reply:
> That's weird, since the link https://microsoft.github.io/syntheseus/stable/ works and points to the correct version.

Yeah, by default navigating to the docs will go to `stable` which will redirect to the version with that alias.

Reply:
I changed it; how about now?